src/run_bpftrace.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include "bpftrace.h"
+
+using namespace bpftrace;
```

Comment:
From the Cpp Core Guidelines: ["Don’t write using namespace at global scope in a header file"](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#sf7-dont-write-using-namespace-at-global-scope-in-a-header-file)

We've either got to fully qualify everything we use (`bpftrace::BPFtrace`) or maybe it'd make sense to just put this whole file in the `bpftrace` namespace as well?